code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -67,7 +67,7 @@
 		return 1
 
 /obj/item/weapon/gun/projectile/revolver/detective45
-	name = "revolver"
+	name = ".45 revolver"

```

Comment:
Honestly, I prefer the ambiguity, but I have dumb opinions.

Reply:
I like the description idea better.

Reply:
the description is already different, and yeah you could just name it "revolver" if you really wanted